backend/src/apiserver/main.go

```diff
@@ -140,7 +141,7 @@ func startHttpProxy(resourceManager *resource.ResourceManager) {
 	topMux.HandleFunc("/apis/v1beta1/pipelines/upload", pipelineUploadServer.UploadPipeline)
 	topMux.HandleFunc("/apis/v1beta1/pipelines/upload_version", pipelineUploadServer.UploadPipelineVersion)
 	topMux.HandleFunc("/apis/v1beta1/healthz", func(w http.ResponseWriter, r *http.Request) {
-		io.WriteString(w, `{"commit_sha":"`+common.GetStringConfigWithDefault("COMMIT_SHA", "unknown")+`", "tag_name":"`+common.GetStringConfigWithDefault("TAG_NAME", "unknown")+`"}`)
+		io.WriteString(w, `{"commit_sha":"`+common.GetStringConfigWithDefault("COMMIT_SHA", "unknown")+`", "tag_name":"`+common.GetStringConfigWithDefault("TAG_NAME", "unknown")+`", "multi_user":`+strconv.FormatBool(common.IsMultiUserMode())+`}`)
```

Comment:
Hi, I just realized there's some complexities here. KFP SDK usually connect to KFP UI server in fact, KFP UI server redirects all api requests to KFP API Server, but the healthz endpoint is not redirected.

So you need to make sure both healthz endpoint expose the multi user option.
You can find UI server's healthz endpoint at https://github.com/kubeflow/pipelines/blob/80e1d7063dc6eea0d17606c26fb61aff5eca4310/frontend/server/handlers/healthz.ts#L81.

Reply:
Thanks for catching that, I wouldn't have found that by myself. I hope this is addressed properly in my latest commit.

Reply:
Thank you @DavidSpek!

First, because right now we support people using KFP SDK with KFP UI/KFP API endpoint interchangeably, so I'd suggest using the same field name for `multi_user` field, you can choose to expose both `apiServerMultiUser` and `multi_user` fields in UI server as a stopgap solution.

Second, would you mind adding an integration test for the `healthz` endpoint for UI server?
You can find existing tests in https://github.com/kubeflow/pipelines/blob/898dff37320d128de393a0146d8b7f1009aff55a/frontend/server/app.test.ts#L115.